scenarios/multi_pow_01/reports/Multi-PoW Mining Simulation Approach.md

```diff
@@ -0,0 +1,85 @@
+# Multi-PoW Mining Simulation Approach
+
+(_The code base is available on GitHub at [**tari-labs/modelling/scenarios/multi_pow_01**](https://github.com/tari-labs/modelling/tree/master/scenarios/multi_pow_01)_)
+
+## Mining
+
+To simulate a constant probability of producing a block at the correct difficulty, an oracle is used that asks each algorithm to produce the next block. Each algorithm runs independently, without any inter-dependencies, using its own difficulty adjustment. Mining is based on the average time it takes to solve a block for a given hash rate and target difficulty (see _**Block Time Estimate**_).
+
+```
+solve_time = (target_difficulty/hash_rate) * gradient_r + self.intercept
+achieved_difficulty = ((solve_time - self.intercept) / self.gradient) * hash_rate
+
+```
+
+If an algorithm did not produce the previous solution, an overhead of 1% of the target solve time is added to the solve time to simulate the inefficiencies of block propagation delay on the network and constructing a new block to solve.
```

Comment:
As discussed, I think what these inefficiencies include and the chosen value of 1% should be substantiated

Reply:
Done